### PR TITLE
Auto poll

### PR DESCRIPTION
**Pull request by MattDHill**

### CODE REVIEW

File: ui/src/app/app.component.ts [hunk 1]
```diff
@@ -206,16 +205,13 @@ export class AppComponent {
           case ConnectionFailure.Network:
             message = 'Phone or computer has no network connection.'
             break
-          case ConnectionFailure.Diagnosing:
-            message = new IonicSafeString('Running network diagnostics <ion-spinner style="padding: 0; margin: 0" name="dots"></ion-spinner>')
-            break
           case ConnectionFailure.Tor:
             message = 'Browser unable to connect over Tor.'
-            link = 'https://docs.start9.com/support/FAQ/setup-faq.html#tor-failure'
+            link = 'https://docs.start9.com/support/FAQ/troubleshooting.html#tor-failure'
```
Comment (Blu-J) [inline ui/src/app/app.component.ts:210]: Currently a 404, are we gonna fix this
image

Comment (MattDHill) [reply ui/src/app/app.component.ts:210]: Yes this is expected at present. All UI links will be fixed before deployment

**Review by Blu-J — APPROVED** [review]

**Review by MattDHill — COMMENTED** [review]
Looks like there are some linting issues. ie use of semi colons and double quotations. While we are planning to begin using prettier and can debate linting practices, for now, please stick with the existing conventions as defined in tslint. Otherwise this look OK to me, though I have not tested it on a live device myself.

**Comment by dr-bonez** [comment]
does this need an interactive rebase or should it be squashed?

**Comment by Blu-J** [comment]
> for now, please stick with the existing conventions as defined in tslint

@DrewAns Could you do the fixes, my system isn't setup for the running `npm lint` (Proof in Pic) nor do I have the tslint setup for vscode. As for the prettier fixes, didn't realize until we already pushed, meant to do most editing on Drew's side for the ui
image

**Pull request merged**